views/guide.lavender

```diff
@@ -138,8 +138,8 @@ block content
             | becomes
             pre
               code.
-                &lt;section id="some_id" class="foo bar" attribute1="foo" 
-                attribute2="bar"&gt;&lt;/section&gt;
+                &lt;section id='some_id' class='foo bar' attribute1='foo' 
+                attribute2='bar'&gt;&lt;/section&gt;
```

Comment:
Is there a reason why we're using &lg;/&gt; instead of just the < / > characters?